zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/Content.java

```diff
@@ -375,6 +379,50 @@ public boolean delete(String uri) throws IOException {
         return false;
     }
 
+    public boolean deleteContentJson(String uri) throws IOException {
+        boolean deleteSuccessful;
+
+        Path pathToDelete = toPath(uri);
+
+        if (!Files.exists(pathToDelete)) {
+            deleteSuccessful = false;
+        } else if (isDataJsonFile(pathToDelete)) {
+            Path parentDir = pathToDelete.getParent();
+            deleteSuccessful = deleteDataJsonFile(parentDir);
+        } else {
+            deleteSuccessful = Files.deleteIfExists(pathToDelete);
+        }
+
+        if (deleteSuccessful) {
+            deleteEmptyParentDirectories(pathToDelete);
+        }
+
+        return deleteSuccessful;
+    }
+
+    /**
+     * Delete files from the specified collection directory. directories and cy_data.json are exluded all other files
+     * will be deleted.
+     */
+    boolean deleteDataJsonFile(Path path) throws IOException {
```

Comment:
Method name again, this method is deleting a directory containing the data json, not just a file.